### DISCUSSION TITLE
Meteorological Data - from GRIB format to EPW

### DISCUSSION THREAD

**Discussion by Soossee** [UMEP-dev/UMEP#503] [General]
Hello everyone,
I was trying to do some SOLWEIG analysis and I am having difficulties finding EPW files for the year 2019. To remedy this, I downloaded the ERA5 meteorological files and got the data in GRIB format. 
However, I cannot figure out how to transform the GRIB data into an EPW format so that I can start the SOLWEIG analysis. 
Does anyone have any ideas or any tutorial to recommend to me?
thank you, any help would be helpful!

**Comment by CsillaVG** [comment]
Hi Soossee,
I am not familiar with the GRIB format (yet), but if you don't mind giving another try at downloading ERA5 data again, with https://www.shinyweatherdata.com/ you could get it converted to epw directly.
I hope it helps.
Csilla
